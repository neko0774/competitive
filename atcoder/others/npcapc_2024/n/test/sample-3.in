6 23
101670804 457970042 521121852 851881468 298366530 271962368
881900040 161849089 409608300 527884448 898980182 538728808
624037110 955334452 644656371 684645088 612630196 365375437
135489465 838789241 374389562 238291227 977400760 496900790
921432977 606711088 740916866 405856539 822796504 19906274
631056816 823924205 719574385 332547421 298321832 594473309
145245098 519214077 131200957 702537460 682007417 304700059
419152148 943797679 19640790 464034522 441355576 454665900
822982080 991406863 249537771 45708817 502700323 683669766
326512291 871373558 410675396 208947517 645163777 622674739
606534945 349193474 895660493 579299011 661372605 314032169
497618687 102217570 426720147 363888279 910628723 894753922
