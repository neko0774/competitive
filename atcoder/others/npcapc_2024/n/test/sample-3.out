18827363
93291123
549166310
727345493
212460686
835043567
382235992
234331494
126083178
340949995
361327462
549134620
481914498
34075195
89718312
945811332
898724999
944812555
123616792
779724718
995912550
188150326
361531843
801483934
